backend/Fusion/Fusion.RestApi/Extensions/MigrateDbExtension.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using Fusion.Infrastructure.Database;
+using Microsoft.EntityFrameworkCore;
+
+namespace Fusion.RestApi.Extensions;
+
+[ExcludeFromCodeCoverage]
+public static class MigrateDbExtension
+{
+   /// <summary>
+   /// Extension method to apply migrations to the database on application startup
+   /// </summary>
+   /// <param name="app"></param>
+   /// <returns></returns>
+   public static void ApplyMigrations(this IApplicationBuilder app)
+   {
+      using IServiceScope serviceScope = app.ApplicationServices.CreateScope();
+
+      using ApplicationDbContext dbContext = serviceScope.ServiceProvider.GetRequiredService<ApplicationDbContext>();
+
+      dbContext.Database.Migrate();
```

Comment:
I would add a check, something like that: 
```C#
if (dbContext.Database.GetPendingMigrations().Any())
            await dbContext.Database.MigrateAsync();
```

Reply:
As i know, by default migrate executes only if there are pending migrations

Reply:
Okay, my mistake